uktrade/enquiry-mgmt-tool
# Format files with flake8

## Description of change

This pull request formats all relevant python files in the app using flake8. This includes removing lots of unused imports, unused variables and spacing errors.

I have also added a flake8 configuration file which adds the following rules:

- Sets the max line length to be 99 - in-line with what PEP8 allows and what we have in data hub API
- Excludes certain files e.g. migrations, settings, pycache which do not benefit from being formatted with flake8

This PR is the first of 3 to bring flake8 into the repository, the next two will:

- Set up a pre-commit hook to run flake8 against new code being committed
- Set up flake8 to run in the circle ci build

## Test instructions

Run flake8 on this branch (`docker exec -it enquiry-mgmt-tool_app_1 flake8`)
Check there are no formatting errors
